src/main/java/frc/robot/subsystems/Intake.java

```diff
@@ -165,10 +173,13 @@ public void burnFlashSparks() {
   /** Sends the deploy motor voltage, needs to be called every cycle */
   private void controlPosition(double positionDeg) {
     deployMotorController.setGoal(positionDeg);
-    double demand = deployMotorController.calculate(deployMotorEncoder.getPosition());
```

Comment:
Don't need the new variables. More importantly we need to only do setGoal once. Otherwise I think the changes didn't do anything